config/v3_4_experimental/schema/ignition.json

```diff
@@ -328,6 +328,12 @@
             },
             "thumbprint": {
               "type": ["string", "null"]
+            },
+            "offlineProvisioning": {
```

Comment:
Do we need this?  It looks as though we could just condition based on whether `advertisement` is provided.

Reply:
Lol, your right; not needed was too fixated on having a switch.